torchgeo/datasets/mmearth.py

```diff
@@ -0,0 +1,505 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""MM-Earth Dataset."""
+
+import json
+import os
+from collections.abc import Callable, Sequence
+from typing import Any, ClassVar, cast
+
+import numpy as np
+import torch
+from torch import Tensor
+
+from .errors import DatasetNotFoundError
+from .geo import NonGeoDataset
+from .utils import Path, lazy_import
+
+
+class MMEarth(NonGeoDataset):
+    """MM-Earth dataset.
```

Comment:
```suggestion
    """MMEarth dataset.
```
Same elsewhere, this is the official dataset name